cli/src/main/java/com/devonfw/tools/ide/log/IdeSubLogger.java

```diff
@@ -43,7 +43,7 @@ default String log(Throwable error, String message) {
 
   /**
    * @return {@code true} if this logger is enabled, {@code false} otherwise (this logger does nothing and all {@link #log(String) logged messages} with be
-   * ignored).
+   * ignored). >>>>>>> main
```

Comment:
Leftover from git merge conflict...
```suggestion
   * ignored).
```